### ISSUE TITLE
gha-creds file getting pushed to the storage

### ISSUE THREAD

**Issue by harshitaXerox** [google-github-actions/upload-cloud-storage#311]
### TL;DR

Hi Team,

The gha-creds* file is also getting pushed to the cloud storage along with the build folder.

Could you please check and let us know if by any chance we can ignore this file.

 - id: 'auth'
        uses: 'google-github-actions/auth@v0'
        with:
           credentials_json: '${{ secrets.***}}'
           export_environment_variables: false
              
      - id: 'upload-folder'
        uses: 'google-github-actions/upload-cloud-storage@v1'
        with:
          path: '.'
          destination: 'carear-metroplex-portal' 
          parent: false
          process_gcloudignore: true   

Thanks
Harshita

### Action YAML

```yaml
name: Move to Firebase storage on workflow_dispatch
'on': 
 workflow_dispatch:
    inputs:
      environment:
        description: 'Environment'
        required: true
        default: 'metroplex'
        
jobs:
  build:
    name: Build
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
        with:
          fetch-depth: 0
          path: main
      - name: Checkout ex builder repo
        uses: actions/checkout@v3
        with:
          repository: carear-io/CareARExperienceBuilder
          path: CareARExperienceBuilder
          ref: ${{ github.event.inputs.experiencebuilderbranch  }} 
          token: ${{ secrets.GIT_ACCESSTOKEN_CICD }}
    
 [[ here we have the logic to build the app]
       
      - name: Upload Artifact
        uses: actions/upload-artifact@master
        with:
          name: build
          path: ./main/build
          
  deploy:
    name: Deploy
    needs: [build]
    runs-on: ubuntu-latest
    steps:
      - name: Download the build output
        uses: actions/download-artifact@v3
        with:
          name: build

      - id: 'auth'
        uses: 'google-github-actions/auth@v0'
        with:
           credentials_json: '${{ secrets.FIREBASE_CAREAR_METROPLEX}}'    

      - id: 'upload-folder'
        uses: 'google-github-actions/upload-cloud-storage@v1'
        with:
          path: '.'
          destination: 'carear-metroplex-portal' 
          parent: false
          process_gcloudignore: true
```

**Comment by sethvargo** [comment]
https://github.com/google-github-actions/auth/blob/main/docs/TROUBLESHOOTING.md#dirty-git-or-bundled-credentials

**Comment by harshitaXerox** [comment]
I tried with google-github-actions/auth@v1 but still the gha-creds file is getting pushed along with the artifact.

**Comment by sethvargo** [comment]
Is it in your .gcloudignore?

**Comment by harshitaXerox** [comment]
yes it is

**Comment by sethvargo** [comment]
Hmm - okay.

- Could you please show your `.gcloudignore` and `.gitignore` for this repo?

- Could you please show the entire repo structure (`tree` on Linux systems)

- Could you please provide the [debug output](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) for the complete GitHub Actions workflow run?

**Comment by harshitaXerox** [comment]
Hi Seth,

Here are all the details that you have asked  for... 

Thanks

![image](https://user-images.githubusercontent.com/82833336/222759041-8924c003-9e86-4082-8918-2f6e1458a532.png)
![image](https://user-images.githubusercontent.com/82833336/222759188-30866df9-cd7c-41ce-a9ec-c91d93c7bd21.png)
![image](https://user-images.githubusercontent.com/82833336/222759285-c85a450b-c631-42c5-ab62-ccbc93d1c893.png)
![image](https://user-images.githubusercontent.com/82833336/222765436-a2d9ebe4-79cf-4143-8874-b7b3628ac4d0.png)

[logs_2047.zip](https://github.com/google-github-actions/upload-cloud-storage/files/10883637/logs_2047.zip)

**Comment by sethvargo** [comment]
Hi @harshitaXerox - those are not the debug logs - there are no debug statements. You need to enable debug logging as shown [here](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) and re-run your workflow.

**Comment by harshitaXerox** [comment]
Hi @sethvargo  here are the debug logs 
[logs_2061.zip](https://github.com/google-github-actions/upload-cloud-storage/files/10932447/logs_2061.zip)

**Comment by harshitaXerox** [comment]
Here are the logs after setting both ACTIONS_RUNNER_DEBUG to true  and ACTIONS_STEP_DEBUG to true.
[logs_2063.zip](https://github.com/google-github-actions/upload-cloud-storage/files/10933463/logs_2063.zip)

**Comment by sethvargo** [comment]
You aren't checking out the repository in the `deploy` step, so the `.gcloudignore` isn't present. Specifically, `$GITHUB_WORKSPACE` is not set, so the action does not know where to look for a gcloud ignore.
